packages/venia-ui/lib/components/CartPage/ProductListing/product.css

```diff
@@ -2,10 +2,20 @@
     align-items: start;
     display: grid;
     grid-gap: 0.5rem 1rem;
-    grid-template-areas: 'image details kebab';
+    grid-template-areas:
+        'error error error'
+        'image details kebab';
     grid-template-columns: 100px 1fr min-content;
 }
 
+.errorText {
+    grid-area: 'error';
+    color: rgb(var(--venia-global-color-error));
+    grid-column-start: 1;
+    grid-column-end: 4;
+    padding-bottom: 0.5rem;
+}
```

Comment:
```suggestion
.errorText {
    grid-area: error;
    color: rgb(var(--venia-global-color-error));
    padding-bottom: 0.5rem;
}
```

Reply:
Removing the column start/end attributes causes the error to wrap within the first column.

Reply:
Oh - I had the wrong syntax for `grid-area` :(

Reply:
https://github.com/magento/pwa-studio/pull/2588/commits/e564fbf6624bfaf678360bc8b580f5b1dec461bf